front/app/component-library/components/Radio/index.tsx

```diff
@@ -17,7 +17,7 @@ const HiddenRadio = styled.input.attrs({ type: 'radio' })`
   ${hideVisually()};
 `;
 
-const CustomRadio = styled.div`
+const CustomRadio = styled.div<{ borderColor?: string }>`
```

Comment:
```suggestion
const CustomRadio = styled.div<{ borderColor: string | undefined }>`
```
Sorry, more nitpicking. Just wanted to highlight that optional is not necessarily the same as `undefined`, since we explicitly set it to undefined. But also not a big deal here.

Reply:
Yep I missed that one - thanks for catching it!